libs/shared/form-fields/src/lib/FieldDescription/FieldDescription.tsx

```diff
@@ -1,17 +1,22 @@
 import React, { FC } from 'react'
 import { Text } from '@island.is/island-ui/core'
 
-const Description: FC<{ description: string }> = ({ description }) => {
+interface Props {
+  description: string
+}
+
+export const FieldDescription: FC<Props> = ({ description }) => {
   const getMarkup = () => {
     return {
       __html: description,
     }
   }
   return (
     <Text marginTop={2} marginBottom={1}>
-      <span dangerouslySetInnerHTML={getMarkup()} />
+      {/* <span dangerouslySetInnerHTML={getMarkup()} /> */}
```

Comment:
hmm, dont we need dangerouslySetInnerHTML?

Reply:
Oops that was from debugging, good catch :) Will fix now.

Reply:
Should be fixed now.